northstar/src/runtime/minijail.rs

```diff
@@ -469,8 +470,12 @@ impl Process {
     /// it is SIGKILLed.
     pub async fn terminate(&mut self, timeout: time::Duration) -> Result<ExitStatus, Error> {
         debug!("Sending SIGTERM to {}", self.pid);
-        signal::kill(unistd::Pid::from_raw(self.pid as i32), Some(SIGTERM))
-            .map_err(|e| Error::Os(format!("Failed to SIGTERM {}", self.pid), e))?;
+
+        match signal::kill(unistd::Pid::from_raw(self.pid as i32), Some(SIGTERM)) {
+            Err(nix::Error::Sys(Errno::ESRCH)) => return (&mut self.exit_status).await,
```

Comment:
`ESRCH` means that libc couldn't find a matching process. Isn't this an error condition? We're trying to terminate a process and it's not found. Must be something wrong in the state.

Reply:
Peter found this issue, this is a race if where the process can exit just before this point in which case its pid is no longer valid. Maybe I should put this as a comment.

Reply:
Ok! Thank for pointing out.

A comment would be great.